planner/core/common_plans.go

```diff
@@ -318,7 +318,7 @@ func (e *Execute) checkPreparedPriv(ctx context.Context, sctx sessionctx.Context
 }
 
 // GetBindSQL4PlanCache used to get the bindSQL for plan cache to build the plan cache key.
-func GetBindSQL4PlanCache(sctx sessionctx.Context, preparedStmt *CachedPrepareStmt) string {
+func GetBindSQL4PlanCache(sctx sessionctx.Context, preparedStmt *CachedPrepareStmt, ignore *bool) string {
```

Comment:
Why not treat the `ignore` as the return value rather than input parameter?

Reply:
thks, i'm adapting golang‘s idiomatic syntax